### PR TITLE
843279-UG document for trackball label template support & trackball group label support

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description**
Added UG document for following content:

1. Trackball Label Template
2. Customize the Axis Label with DataTemplate
3. Trackball group label support  - GroupAllPoints
4. ShowLabel in TrackballBehavior

### CODE REVIEW

File: MAUI/Cartesian-Charts/Trackball.md [hunk 1]
```diff
@@ -220,6 +221,189 @@ this.Content = chart;
 
 {% endtabs %}
 
+### Trackball Label Template
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [ChartSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartSeries.html).
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:226]: @EzhilarasanElangovan31 TrackballLabelTemplate exposed in Cartesian Series not Chart series. Modify it

File: MAUI/Cartesian-Charts/Trackball.md [hunk 2]
```diff
@@ -220,6 +221,189 @@ this.Content = chart;
...
+    };
+
+    horizontalStackLayout.Add(label);
+    horizontalStackLayout.Add(image);
+
+    return horizontalStackLayout;
+});
+
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:313]: @EzhilarasanElangovan31  series name not updated correctly

File: MAUI/Cartesian-Charts/Trackball.md [hunk 3]
```diff
@@ -220,6 +221,189 @@ this.Content = chart;
...
+        FontSize = 15
+    };
+    label.SetBinding(Label.TextProperty, "ValueX");
+
+    horizontalStackLayout.Add(label);
+
+    return horizontalStackLayout;
+});
+
+primaryAxis.TrackballLabelTemplate = trackballLabelTemplate;
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:390]: @EzhilarasanElangovan31 template name wrongly updated

File: MAUI/Cartesian-Charts/Trackball.md [hunk 4]
```diff
@@ -220,6 +221,189 @@ this.Content = chart;
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+            <DataTemplate x:Key="axisLabelTemplate">
+                <HorizontalStackLayout>
+                    <Label WidthRequest="50" HeightRequest="20" HorizontalTextAlignment="Center" BackgroundColor="Blue" Text="{Binding ValueX}" TextColor="White" FontSize ="15"/>
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:339]: @EzhilarasanElangovan31 Only Label used in HorizontalStackLayout for template. Why need to use HorizontalStackLayout? You can directly use Label alone

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 In ChartTrackballBehavior, we are also exposing ShowLabel bool property. Add content for it also

File: MAUI/Cartesian-Charts/Trackball.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md:226]: Modified to Cartesian Series

File: MAUI/Cartesian-Charts/Trackball.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md:313]: Updated series name

File: MAUI/Cartesian-Charts/Trackball.md (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Updated the template name

File: MAUI/Cartesian-Charts/Trackball.md (hunk 4 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Removed HorizontalStack layout

**Comment by EzhilarasanElangovan31** [comment]
> @EzhilarasanElangovan31 In ChartTrackballBehavior, we are also exposing ShowLabel bool property. Add content for it also
@VimalaThirumalaiKumarSF3739  added ShowLabel bool property content.

File: MAUI/Cartesian-Charts/Trackball.md [hunk 5]
```diff
@@ -220,6 +222,183 @@ this.Content = chart;
...
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+        <DataTemplate x:Key="trackballLabelTemplate">
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:235]: @EzhilarasanElangovan31 update template from SB, use person image instead of Path

Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md:235]: Modified

File: MAUI/Cartesian-Charts/Trackball.md [hunk 6]
```diff
@@ -220,6 +222,206 @@ this.Content = chart;
...
+                <Label Text="{Binding Label,StringFormat='{0}M'}" FontSize="15" HorizontalOptions="Center" TextColor="White" />
+            </HorizontalStackLayout>         
+        </DataTemplate>
+    </chart:SfCartesianChart.Resources>
+
+   <chart:SfCartesianChart.TrackballBehavior>
+      <chart:ChartTrackballBehavior/>
+   </chart:SfCartesianChart.TrackballBehavior>
+    . . .
+    <chart:LineSeries ItemsSource="{Binding Data1}"
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Trackball.md:249]: @EzhilarasanElangovan31  Label property not assigned in series

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 in image, used starting letter as lower case for person name. Modify it. Also, rebase your branch and work

File: MAUI/Cartesian-Charts/Trackball.md (hunk 6 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md:249]: Added label property in series

File: MAUI/Cartesian-Charts/Trackball.md [hunk 7]
```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
 
 {% endtabs %}
 
+### Trackball Label Template
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
```
[the remark this answers is not in the document]
Comment (Saravanan-Madhesh) [reply MAUI/Cartesian-Charts/Trackball.md:265]: @EzhilarasanElangovan31  have you used ChatGpt, for content correction?

File: MAUI/Cartesian-Charts/Trackball.md [hunk 8]
```diff
@@ -52,6 +53,7 @@ The [DisplayMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.Cha
 
 * `FloatAllPoints` – Displays labels for all the data points along the vertical line.
 * `NearestPoint` – Displays label for a single data point nearer to the touch point on the chart area.
+* `GroupAllPoints` – Displays label for all the data points grouped and positioned at the top of the chart area.
```
[the remark this answers is not in the document]
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Modified the content

File: MAUI/Cartesian-Charts/Trackball.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Modified the content

[the remark this answers is not in the document]
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Modified the content, Yes sir, used ChatGPT.

File: MAUI/Cartesian-Charts/Trackball.md [hunk 9]
```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
 
 {% endtabs %}
 
+### Trackball Label Template
+
+Trackball labels can be customized by adding any view as its template by using [TrackballLabelTemplate]() of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
+
+T> The `DataItem` can be used to access the data linked to the associated business model
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Trackball.md:267]: Mention what is binding context for label template.

File: MAUI/Cartesian-Charts/Trackball.md [hunk 10]
```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
...
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Trackball Label Template support in MAUI chart](Trackball_images/maui_chart_trackball_label_template.png)
+
+## Customize the Axis Label with DataTemplate
+
+Customize the appearance of axis label of trackball using [TrackballLabelTemplate[]() property of [ChartAxis](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html).
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Trackball.md:393]: @EzhilarasanElangovan31 , content is not good, and the grammar is also incorrect. have you checked previous xamarin UG?

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]
Address the comments.

File: MAUI/Cartesian-Charts/Trackball.md [hunk 11]
```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
...
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![Trackball Label Template support in MAUI chart](Trackball_images/maui_chart_trackball_label_template.png)
+
+## Customize the Axis Label with DataTemplate
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Trackball.md:392]: Trackball axis label template - change title.

File: MAUI/Cartesian-Charts/Trackball.md (hunk 9 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Mentioned the binding context

File: MAUI/Cartesian-Charts/Trackball.md (hunk 11 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md:392]: Changed the title

File: MAUI/Cartesian-Charts/Trackball.md (hunk 10 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Trackball.md]: Modified the content and checked grammar also.

**Comment by EzhilarasanElangovan31** [comment]
@Saravanan-Madhesh sir, Addressed the comments.

**Review by Saravanan-Madhesh — APPROVED** [review]
Approved as discussed.

**Comment by Saravanan-Madhesh** [comment]
@VimalaThirumalaiKumarSF3739  add gif for trackball template.

**Pull request merged**